acm/src/main/java/org/literacybridge/acm/cloud/ProjectsHelper.java

```diff
@@ -33,20 +33,21 @@ public class ProjectsHelper {
      * Each of the sub-lists is the same thing, preferred name first.
      */
     public static final String[][] PROGSPEC_PREFERED_NAMES = new String[][] {
-        // New, old files
-        new String[] {"pub_recipients.csv", "recipients.csv"},
-        new String[] {"pub_deployments.csv", "deployment_spec.csv"},
-        new String[] {"pub_content.csv", "content.csv"},
-        // Added files
-        new String[] {"pub_general.csv"},
-        new String[] {"pub_progspec.xlsx"},
-        // Obsolete file
-        new String[] {"recipients_map.csv"},
+            // New, old files
+            new String[] { "pub_recipients.csv", "recipients.csv" },
+            new String[] { "pub_deployments.csv", "deployment_spec.csv" },
+            new String[] { "pub_content.csv", "content.csv" },
+            new String[] { "pub_languages.csv", "languages.csv" },
```

Comment:
"pub_languages.csv" is new, right?, so there are not old files with the wrong name. In that case, we don't need "languages.csv", because it can't exist.